.github/workflows/release.yml

```
@@ -46,10 +51,37 @@ jobs:
           echo "::set-output name=name::${GITHUB_REF#refs/*/}"
           echo "::set-output name=version::${GITHUB_REF##*/}"
         id: extract_branch
+      - name: Check npm version preid
+        run: |
+          HAS_PREID=$(node -p "'${{ steps.extract_branch.outputs.version }}'.startsWith('prerelease-') ? 1 : 0")
```

[nits]

```yml
run: |
  ...
env:
  VERSION: ${{ steps.extract_branch.outputs.version }}
```

としてoutputsを環境変数に入れて

```suggestion
          HAS_PREID=$(node -p "process.env.VERSION.startsWith('prerelease-') ? 1 : 0")
```

環境変数としてアクセスした方が短くなるし読みやすそうだな〜とは思いました。

---

https://github.com/openameba/cwv-logger/pull/8/commits/87e196d8705392aeeeefc8ac95925d57877471d9
こちらで修正しました:pray: